packages/EasyCodingStandard/src/Application/Application.php

```diff
@@ -112,8 +120,19 @@ private function processFoundFiles(array $files): void
             // add it elsewhere
             $this->changedFilesDetector->addFile($relativePath);
 
-            $this->fixerFileProcessor->processFile($fileInfo);
-            $this->sniffFileProcessor->processFile($fileInfo);
+            try {
+                $this->fixerFileProcessor->processFile($fileInfo);
+                $this->sniffFileProcessor->processFile($fileInfo);
+            } catch(\ParseError $e) {
```

Comment:
Same as above, `ParserError` = `$parseError`. 

$e and similar [shortcut naming](https://github.com/object-calisthenics/phpcs-calisthenics-rules#6-do-not-abbreviate) leads to unneeded thinking about the property.